.clang-format

```diff
@@ -0,0 +1,108 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
```

Comment:
The real reason is that it was autogenerated this way. But I guess I can either remove it completely or leave it active. I'll take another look at this config.